### DISCUSSION TITLE
Is there a way to set default options?

### DISCUSSION THREAD

**Discussion by thepuzzlemaster** [toomuchdesign/next-page-tester#289] [Q&A]
I'm curious if there is a way to set an option that is used by default in all `getPage` calls, unless overwritten by an explicit option value passed in.

My use-case is that I'd like all my `getPage` calls to use the same `.env.local` file for pulling in my environment variables. But it seems redundant to have to add that into every call to `getPage`. I didn't see anything in the docs to indicate something like this is supported.

Does this exist? And if not, any reason it couldn't be added?

Thanks!

**Comment by toomuchdesign** [comment]
The suggested way to use `next-page-tester` is through a wrapping function which calls `getPage` method with default settings.
